runtime/doc/vim_diff.txt

```diff
@@ -85,6 +85,10 @@ of these in your config by simply removing the mapping, e.g. ":unmap Y".
 	nnoremap <C-L> <Cmd>nohlsearch<Bar>diffupdate<CR><C-L>
 	inoremap <C-U> <C-G>u<C-U>
 	inoremap <C-W> <C-G>u<C-W>
+	nmap ]g <Plug>(VimDiagnosticGotoNext)
+	nmap [g <Plug>(VimDiagnosticGotoPrev)
+	nmap ]G <Plug>(VimDiagnosticGotoLast)
+	nmap [G <Plug>(VimDiagnosticGotoFirst)
```

Comment:
Hmm there is no connection from these plug mappings now to the help docs associated with what they do currently.

`<Plug>(VimDiagnosticGotoNext)` -> `vim.diagnostic.goto_next {}`.

I think perhaps they should be mentioned in the `*diagnostic-maps*` section.

It may even make more sense just to only make these `Plug` mappings and not actually map to any physical keyboard buttons, but I'm open to other people's opinions on that. We should at least link them in the help docs (in case they change in the future)

Reply:
> It may even make more sense just to only make these Plug mappings and not actually map to any physical keyboard buttons, but I'm open to other people's opinions on that. We should at least link them in the help docs (in case they change in the future)

Personally I think this is only valuable if we do actually make default physical mappings. I'm not sure what the value add of providing default `<Plug>` mappings is since it's an equal burden on the user to create a `<Plug>` vs `<Cmd>` mapping, but the latter allows them to customize to their preferences.

If the consensus is that we don't want to add any new default mappings then I am fine with closing this.

Reply:
Well `<Plug>` mapping is "future-proof", as in, if we change the module location, file path, naming scheme, default arguments, etc., the `<Plug>` mapping can evolve separately.

But I'm happy to put the default ones in :) It seems good to me. They just need to be linked in the docs.